docs/CONFIG.md

```diff
@@ -59,14 +59,21 @@ When you have an Enterprise Search deployment on Elastic Cloud post 8.5.0, the c
 
 ### Run the connector service for a custom connector
 
-1. Go to Kibana, _Enterprise Search_ > _Create an Elasticsearch index_. Use the `Build a connector` option for an ingestion method to create an index.
+1. Go to Kibana, _Enterprise Search_ > _Create an Elasticsearch index_ > _Connector_. Choose `Customized connector` and continue to create an index.
 2. Create an API key to work with the connector. It should be done using the `Generate API key` button under `Configuration` tab.
 3. Configure your connector service application. You need to configure the followings fields, and leave the rest as default.
    1. `elasticsearch.host`: Configure this to the Elasticsearch endpoint.
    2. `elasticsearch.api_key`: Configure the API key generated in step 2. Make sure `elasticsearch.username` is not configured.
    3. `connector_id`: You can find the `connector_id` in step 3 `Deploy a connector` under `Configuration` tab in Kibana.
    4. `service_type`: Configure it to the service type of your new connector.
+   5. Leave `native_service_types` empty, unless you want to sync any native connectors with this connector service instance. In such a case, make sure the API key configured in step 2 has the proper privileges, or you can choose to use basic auth (`username` and `password`) with the proper privileges.
 4. Run the connector service application with
     ```shell
     make run
     ```
+
+### Run the connector service for a converted native connector
+
+1. Follow the instructions in [Run the connector service in native mode](#run-the-connector-service-in-native-mode) to create a native connector.
+2. Go to the `Configuration` tab and convert it to a customized connector via `Convert connector`.
+3. Follow the instructions in [Run the connector service for a custom connector](#run-the-connector-service-for-a-custom-connector) to run the connector service.
```

Comment:
Nice, thanks for adding this section!  Are we saying that regardless of whether they are customizing the native connector or leaving it unmodified after conversion, they must treat it like a custom connector?  Or should we split this up into two sections, 1 for converted native connector (unmodified) v.s. converted native connector (customized)?

Reply:
Sorry I have totally overlooked this issue.

>Are we saying that regardless of whether they are customizing the native connector or leaving it unmodified after conversion, they must treat it like a custom connector?

Yes, as long as it's not a native connector, it should be treated as a custom connector and only run by an on-prem connector service.

With the [work](https://github.com/elastic/connectors-python/pull/1063) done by Sean, I will remove the section to run connector service on-prem in native mode.